test/test.transporters.js

```diff
@@ -124,4 +124,22 @@ it('should return errors within response body as instances of Error', function(d
       done();
     });
   });
+
+  it('should handle 5xx responses that include errors', function(done) {
+    var google = require('../lib/googleapis');
+    var scope = nock('https://www.googleapis.com')
+      .post('/urlshortener/v1/url')
+      .reply(500, { error: {message: 'There was an error!'}});
```

Comment:
Isn't the whole body the entire object I put above? Not just the body component of that object? I might be mistaken.